internal/chezmoi/findexecutable_darwin_test.go

```diff
@@ -0,0 +1,90 @@
+//go:build darwin
```

Comment:
I _think_ this line is not needed as presence of `darwin` in the filename means that Go will only build it on macOS.

Reply:
I’ll give it a try as its own commit to see if there are build issues. That makes it easy to revert if there are.